constellations/clients/orionFakeDataFiller/filler.go

```diff
@@ -228,28 +238,87 @@ func runFiller(hostAddress string) {
 	)
 
 	// Create sessions
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_1",
+		"ap_java_2020_fall_class1",
 		"false",
+		3,
 	)
 
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_2",
+		"ap_calculus_2020_fall_class1",
 		"true",
+		2,
+	)
+
+	// Create Accounts
+	createAccount(
+		hostAddress,
+		"emailaddress1@gmail.com",
+		"jhdgjhddjhdjuj",
+	)
+	createAccount(hostAddress,
+		"emailaddress2@gmail.com",
+		"2redssssa",
+	)
+
+	// Create Users
+	createUser(
+		hostAddress,
+		"Joe",
+		"Smith",
+		"",
+		"emailaddress1@gmail.com",
+		"301-123-4567",
+		0,
+		1,
+		"notes1",
+		"schoolone",
+		2001,
+	)
```

Comment:
can you make 2 users for account1. Account2 having only one user is fine too. Try to simulate what it will look like in "real life".

So account1 is for one family, while account2 is for another family.

Reply:
Account1 could have a parent and 2 kids or something? And if they're a guardian, they shouldn't have a school / grad year